### DISCUSSION TITLE
Event triggered when you create a link from one node to another and it remains 'linked'.

### DISCUSSION THREAD

**Discussion by JuanMaGMZG** [Blazor-Diagrams/Blazor.Diagrams#343] [General]
In the new version and with the new names (Pointer), what would be the event that captures that a link has been created from a source node to a target node?

Thank you very much for everything.

**Comment by zHaytam** [comment]
Hello,

It's still the `Diagram.Links.Added` event, which tells you that a new link has been created.  
Then you can hook into `theNewLink.TargetAttached` to know when the link got attached.

**Reply by JuanMaGMZG** [reply]
Thank you very much for your response,

Great, I have it now, but I don't see a way to obtain the source node and the target node.

It says here: model.Source.Model.Parent.Id

but Parent is giving me an error saying that ILinkable does not contain a definition for Parent.

**Reply by zHaytam** [reply]
Check https://github.com/Blazor-Diagrams/Blazor.Diagrams/issues/342

**Reply by JuanMaGMZG** [reply]
Great! Thx
